packages/ui-components/src/Icon/Icon.module.scss

```diff
@@ -1,68 +1,62 @@
 @import "../styles/tokens.scss";
-
 .icon {
-  display: inline-flex;
+  fill: $color-core-neutral-7;
   margin: auto;
 
   &:disabled {
     pointer-events: none;
   }
 }
 
-.size-xx-large {
-  height: 80px;
-  width: 80px;
-}
-
 .size-x-large {
-  height: 56px;
-  width: 56px;
-}
-
-.size-large {
-  height: 48px;
-  width: 48px;
-}
-
-.size-medium {
   height: 32px;
   width: 32px;
 }
 
-.size-default {
+.size-large {
   height: 24px;
   width: 24px;
 }
 
+.size-medium {
+  height: 20px;
+  width: 20px;
+}
+
+.size-default,
 .size-small {
   height: 16px;
   width: 16px;
 }
 
-.tint-blue {
+.size-tiny {
+  height: 8px;
+  width: 8px;
+}
+
+.intent-info {
   fill: $color-base-blue;
 }
 
-.tint-green {
+.intent-primary {
+  fill: $color-base-purple;
+}
+
+.intent-success {
   fill: $color-base-green;
 }
 
-.tint-red {
+.intent-danger {
   fill: $color-base-red;
 }
 
-.tint-orange {
+.intent-warning {
   fill: $color-base-orange;
 }
 
-.tint-white {
```

Comment:
Is this one meant to be the "Knockout" color in Figma (when rendered on a dark background)?